src/math-functions.js

```diff
@@ -64,15 +64,15 @@ export function multiplyAnyArray(dynamicArray) { //eslint-disable-line
     let returnProduct = 1;
     let returnString = 'The numbers ';
 
-    if (dynamicArray.length % 2 === 0) {
-        for (let i = 0; i < dynamicArray.length; i+=2){
-            returnProduct *= multiply(dynamicArray[i], dynamicArray[i+1])[0];
-            returnString += dynamicArray[i] + ',' + dynamicArray[i+1] + ',';
+    if(dynamicArray.length % 2 === 0) {
+        for(let i = 0; i < dynamicArray.length; i += 2){
+            returnProduct *= multiply(dynamicArray[i], dynamicArray[i + 1])[0];
+            returnString += dynamicArray[i] + ',' + dynamicArray[i + 1] + ',';
         }
-    } else {
-        for (let i = 0; i < dynamicArray.length -1; i+=2){
-            returnProduct *= multiply(dynamicArray[i], dynamicArray[i+1])[0];
-            returnString += dynamicArray[i] + ',' + dynamicArray[i+1] + ',';
+    }else{
+        for(let i = 0; i < dynamicArray.length -1; i+=2){
+            returnProduct *= multiply(dynamicArray[i], dynamicArray[i + 1])[0];
+            returnString += dynamicArray[i] + ',' + dynamicArray[i + 1] + ',';
```

Comment:
Totally a clever solution! Great job puzzling this out. You can use `${dynamicArray}` as well.